save-frontend/src/main/kotlin/org/cqfn/save/frontend/components/views/CreationView.kt

```diff
@@ -109,22 +107,34 @@ enum class GitConnectionStatusEnum {
 @OptIn(ExperimentalJsExport::class)
 class CreationView : AbstractView<Props, ProjectSaveViewState>(true) {
     private val fieldsMap: MutableMap<InputTypes, String> = mutableMapOf()
+    private val selectFormRequired = selectFormRequired(
+        onChangeFun = ::changeFields,
+    )
 
     init {
         state.isErrorWithProjectSave = false
         state.errorMessage = ""
         state.gitConnectionCheckingStatus = GitConnectionStatusEnum.NOT_CHECKED
 
-        state.isValidOwner = true
+        state.isValidOrganization = true
         state.isValidProjectName = true
         state.isValidGitUrl = true
         state.isValidGitUser = true
         state.isValidGitToken = true
     }
 
-    private fun changeFields(fieldName: InputTypes, target: Event, isProject: Boolean = true) {
-        val tg = target.target as HTMLInputElement
-        if (isProject) fieldsMap[fieldName] = tg.value else fieldsMap[fieldName] = tg.value
+    private fun changeFields(
+        fieldName: InputTypes,
+        target: Event,
+        isProject: Boolean = true,
+        isHtmlInputElement: Boolean = true
+    ) {
+        val tg = if (isHtmlInputElement) {
```

Comment:
Why not use `when` with `is` checks here?

Reply:
Fixed